docs/user-guide/options.md

```diff
@@ -36,7 +36,11 @@ CLI flag: `--fix`
 
 Automatically fix, where possible, problems reported by rules.
 
-For CSS with standard syntax, Stylelint uses [postcss-safe-parser](https://www.npmjs.com/package/postcss-safe-parser) to fix syntax errors.
+Options are:
+
+- `true` or `lax` - uses [postcss-safe-parser](https://www.npmjs.com/package/postcss-safe-parser) to fix as much as possible, even when there are syntax errors
+- `strict` - uses [PostCSS Parser](https://postcss.org/api/#postcss-parser) and only fixes problems when there are no syntax errors
+- `false` (default) - does not fix any problems
 
```

Comment:
[suggestion] This is a just idea and not required, but what about adding the following tip?

> [!TIP]
> If you don't care about strictness now, we recommend using `true` because `"strict"` may become the default in the future.

```suggestion

> [!TIP]
> If you don't care about strictness now, we recommend using `true` because `"strict"` may become the default in the future.

```

Reply:
refs
stylelint/stylelint.io#409
https://github.com/incentro-ecx/remark-github-admonitions-to-directives#%EF%B8%8F-customizing-the-mapping